packages/nuejs.org/docs/installation.md

```diff
@@ -24,48 +24,44 @@ Create your first Nue website:
 
 
 ```sh
-# create an empty directory and cd to it
-mkdir "simple-blog" && cd "$_"
```

Comment:
No mkdir step needed?

Reply:
Is included in create:

https://github.com/nuejs/nue/blob/e2119f3821764184745a56363acb43f4c77f18c4/packages/nuekit/src/create.js#L37

Is used to prevent different `root`s from failing.

(On a side note: `cd $_` does not work on windows, `&&` also not. I've tested it a few days ago)

Reply:
Awesome. In what case(s) `mkdir` gets called?

The root defaults to "."

https://github.com/nuejs/nue/blob/e2119f3821764184745a56363acb43f4c77f18c4/packages/nuekit/src/cli.js#L85

Reply:
nope. changed that. the create command gets the `root` variable, not the `args.root` one that is used by everything else but create. `root` is `null`.

`mkdir` gets called, when `root` does not exist. see the `if` to the `else` from before:

https://github.com/nuejs/nue/blob/e2119f3821764184745a56363acb43f4c77f18c4/packages/nuekit/src/create.js#L27

Reply:
I see, but in what scenarios the root does not exist when I'm using the command line?

Reply:
I guess the site gets created to a new directory called `simple-blog` unless explicitly specified with `nue create --root my-website`?

Reply:
yes. and if the directory already exists, it checks, whether it is empty.

Reply:
> I see, but in what scenarios the root does not exist when I'm using the command line?

If it hasn't been created by hand beforehand, it gets created.

Reply:
Cool. I'll add a note on this to the installation doc. Merging this now.